common/models/user.js

```diff
@@ -869,9 +1003,21 @@ module.exports = function(User) {
     );
 
     UserModel.remoteMethod(
+      'verify',
+      {
+        description: 'Trigger user\'s identity verification.',
+        accepts: [
+          {arg: 'uid', type: 'string', required: true, http: {source: 'form'}},
+          {arg: 'options', type: 'object', http: 'optionsFromRequest'},
+        ],
+        http: {verb: 'post', path: '/verify'},
+      }
+    );
+
+    UserModel.remoteMethod(
       'confirm',
       {
-        description: 'Confirm a user registration with email verification token.',
+        description: 'Confirm a user registration with identity verification token.',
```

Comment:
although this is the only user's identity verification option implemented as of today, the verify options is not meant to be used only to verify user's email. It could verify user's phone number for example (link sent via SMS). IMO the appropriate wording is identity verification token.